apis/validator/sync_committee_selections.yaml

```
@@ -8,8 +8,9 @@ post:
     used to exchange partial selection proof slot signatures for combined/aggregated selection proofs to 
     allow a validator client to correctly determine if one of its validators has been selected to perform a 
     sync committee contribution (sync aggregation) duty in a slot. Validator clients must query this endpoint
-    at the start of an epoch for all validators that have sync committee contribution duties in that epoch. 
-    Consensus clients need not support this endpoint and may return a 501.
+    at the start of an epoch for the current and lookahead (next) epochs for all validators that have
+    sync committee contribution duties in the current and lookahead epochs. Consensus clients need not support this 
```

So would I be write in saying we change this to:
```suggestion
    each slot for validators that are members of a sync committee.
Consensus clients need not support this 
```

---

yeah, that's correct. 

```
...
Validator clients must query this endpoint at the start of each slot for all validators that are included in the current sync committee.
...
```